py_ts_interfaces/parser.py

```diff
@@ -118,7 +118,7 @@ def helper(
             # We will have to assume it is a valid forward reference now and
             # then just double check that it does indeed reference another
             # Interface class as a post-parse step.
-            type_value = TYPE_MAP.get(node.name, PossibleInterfaceReference(node.name))
+            type_value = TYPE_MAP.get(node.name.lower(), PossibleInterfaceReference(node.name))
```

Comment:
Looks like the `flake8` linter is failing this line for being too long.  If you'd like, here are some instructions for running the tests locally before submitting an update:  https://github.com/cs-cordero/py-ts-interfaces/blob/master/CONTRIBUTING.md#running-tests